website/docs/language/settings/backends/gcs.mdx

```diff
@@ -130,4 +130,5 @@ The following configuration options are supported:
   used when reading and writing state files in the bucket.
   Format should be `projects/{{project}}/locations/{{location}}/keyRings/{{keyRing}}/cryptoKeys/{{name}}`. 
   For more information, including IAM requirements, see [Customer-managed Encryption 
-  Keys](https://cloud.google.com/storage/docs/encryption/customer-managed-keys).
\ No newline at end of file
+  Keys](https://cloud.google.com/storage/docs/encryption/customer-managed-keys).
+- `storage_custom_endpoint` / `GOOGLE_BACKEND_STORAGE_CUSTOM_ENDPOINT` / `GOOGLE_STORAGE_CUSTOM_ENDPOINT` - (Optional) A URL containing three parts: the protocol, the DNS name pointing to a Private Service Connect endpoint, and the path for the Cloud Storage API (`/storage/v1/b`, [see here](https://cloud.google.com/storage/docs/json_api/v1/buckets/get#http-request)). You can either use [a DNS name automatically made by the Service Directory](https://cloud.google.com/vpc/docs/configure-private-service-connect-apis#configure-p-dns) or a [custom DNS name](https://cloud.google.com/vpc/docs/configure-private-service-connect-apis#configure-dns-default) made by you. For example, if you create an endpoint called `xyz` and want to use the automatically-created DNS name, you should set the field value as `https://storage-xyz.p.googleapis.com/storage/v1/b`. For help creating a Private Service Connect endpoint using Terraform, [see this guide](https://cloud.google.com/vpc/docs/configure-private-service-connect-apis#terraform_1).
```

Comment:
Not sure whether to have the user provide a value that contains the full `protocol` + `DNS name` + `/storage/v1/b` or only ask for `DNS name` as the value for this field. I think the protocol would always be https, and I don't expect the path will change anytime soon...

Reply:
I lean toward the whole thing, but don't have a strong preference either way :)